app/javascript/components/ansible-repository-form/ansible-repository-form.schema.js

```diff
@@ -38,15 +39,10 @@ function createSchema(repositoryId) {
       id: 'scm_url',
       name: 'scm_url',
       isRequired: true,
-      validate: [{
-        type: validatorTypes.REQUIRED,
```

Comment:
Do we need to keep required?

Reply:
Likely yes, I don't know of a case when the url wont be required.